masked-number/assets/index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Get started with your Twilio Functions!</title>
+    <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Roboto:300,300italic,700,700italic">
+    <link rel="stylesheet" href="https://unpkg.com/normalize.css/normalize.css">
+    <link rel="stylesheet" href="https://unpkg.com/milligram/dist/milligram.min.css">
+    <style>
+      body {
+        padding: 20px;
+        display: flex;
+        flex-direction: column;
+        min-height: 100vh;
+      }
+
+      div[role="main"] {
+        flex: 1;
+      }
+
+      footer {
+        text-align: center;
+      }
+
+      footer p {
+        margin-bottom: 0;
+      }
+
+      #twilio-logo {
+        width: 50px;
+        height: 50px;
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <h1>Welcome to your new Twilio App</h1>
+    </header>
+    <div role="main">
+      <section>
+        <h3>Get started with your app</h3>
+        <p>This app masks your phone number for SMS messages by relaying them through a Twilio phone number. Follow the steps below to test your app:</p>
+        <ol>
+          <li>Text your Twilio phone number with a recipient phone number, a <code>:</code>, and a message.</li>
+          <li>Your recipient should receive your message from your Twilio phone number.</li>
+          <li>Give your Twilio phone number to another person when you want to message them without exposing your personal phone number.</li>
```

Comment:
I think this step could also use some more clarity, we should probably continue a bit with from the standpoint of the other device that received that text from the twilio number, to reply back to it and notice the response on the original phone number